src/z3c/dependencychecker/dependencychecker.py

```diff
@@ -84,6 +84,57 @@
 ['\"]        # Single or double quote.
 """, re.VERBOSE)
 
+FTI_BEHAVIOR_PATTERN = re.compile(r"""
+\s           # Whitespace.
+<element     #
```

Comment:
I think the problem is that we don't know in which context the `<element>` tag appears.
Regex is not really useful for parsing context-aware things such as XML, at least when the context gets relevant as it is the case here (seeh the [Chomsky hierarchy](https://en.wikipedia.org/wiki/Chomsky_hierarchy)).

Maybe it would be better to use a XML parser, such as lxml or minidom, what do you think?